lgc/patch/FragColorExport.cpp

```diff
@@ -692,8 +694,29 @@ CallInst *LowerFragColorExport::AddExportForGenericOutputs(Function *fragEntryPo
   // color format in the metadata.
   m_pipelineState->getPalMetadata()->addColorExportInfo(info);
 
-  // TODO: The export values should be returned.
-  return nullptr;
+  // The export values should be returned.
+  // Update the return type of the function
+  auto retInst = fragEntryPoint->back().getTerminator();
```

Comment:
Same comment as before about relying on there being exactly one ret at the end of the function.

Reply:
The old code way written in such a way that it would not have worked if there were multiple return statements, so I think it is safe for me to assume the same.